roles/container-engine/cri-o/vars/v1.31.yml

```diff
@@ -0,0 +1,13 @@
+---
+crio_conmon: "{{ bin_dir }}/conmon"
+
+# cri-o binary files
+crio_bin_files:
+  - conmon
+  - conmonrs
+  - crun
+  - runc
+  - crio
+  - pinns
+
+crio_status_command: crio status
```

Comment:
HI @philipsabri 

The file is the same as https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/cri-o/vars/v1.29.yml. 

It may does not required.

Reply:
@yankay, this file is not exactly same, `crio_bin_files` is difference.

Reply:
Thanks
/approve